### PR TITLE
Update meta-pelux to rocko

### PR DESCRIPTION
**Pull request by tobsan**
This adds some changes needed to adapt to the rocko release of poky for meta-pelux.

### CODE REVIEW

File: conf/variant/common/bblayers.conf [hunk 1]
```diff
@@ -3,12 +3,11 @@ LCONF_VERSION = "7"
 BBFILES  ?= ""
 BBLAYERS ?= "                                         \
   ${BSPDIR}/sources/poky/meta                         \
-  ${BSPDIR}/sources/poky/meta-yocto                   \
+  ${BSPDIR}/sources/poky/meta-poky                    \
   ${BSPDIR}/sources/poky/meta-yocto-bsp               \
   ${BSPDIR}/sources/meta-openembedded/meta-oe         \
   ${BSPDIR}/sources/meta-openembedded/meta-networking \
   ${BSPDIR}/sources/meta-openembedded/meta-python     \
-  ${BSPDIR}/sources/meta-openembedded/meta-ruby       \
```
Comment (GordanM) [inline conf/variant/common/bblayers.conf]: I'm glad we are getting rid of this but is there an explanation why?

Comment (tobsan) [reply conf/variant/common/bblayers.conf]: The explanation is that `meta-ruby` contained very few recipes, so those that were there were moved somewhere else and the layer was deleted.

Comment (GordanM) [reply conf/variant/common/bblayers.conf]: Okay, perfect so it's not us stopping to use it but the layer content has been moved to other places. Could that be reflected in the commit message or do you think it is redundant?

Comment (tobsan) [reply conf/variant/common/bblayers.conf]: No, it should be mentioned. Will update.

Comment (tobsan) [reply conf/variant/common/bblayers.conf]: Done

**Comment by GordanM** [comment]
Thanks, I'm good!

**Pull request merged**